### PR TITLE
Public message feed filters by level

### PR DESCRIPTION
**Pull request by miaboloix**

### CODE REVIEW

File: src/main/java/com/google/codeu/data/Datastore.java [hunk 1]
```diff
@@ -128,7 +130,8 @@ public User getUser(String email) {
             return null;
         }
         String aboutMe = (String) userEntity.getProperty("aboutMe");
-        User user = new User(email, aboutMe);
+        long level = (long) userEntity.getProperty("level");
+        User user = new User(email, aboutMe, (int) level);
```
Comment (Rachelware) [inline src/main/java/com/google/codeu/data/Datastore.java:134]: Could you explain to me the changing of level's datatype between int and long?

File: src/main/java/com/google/codeu/servlets/MessageFeedServlet.java [hunk 2]
```diff
@@ -26,17 +29,27 @@ public void init(){
...
       for all users. */
     @Override
-	public void doGet(HttpServletRequest request, HttpServletResponse response)
-		
-	    throws IOException{
-
-	        response.setContentType("application/json");
+	public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException{
+
+    	response.setContentType("application/json");
```
Comment (Rachelware) [inline src/main/java/com/google/codeu/servlets/MessageFeedServlet.java:33]: I think indentation is off for this line

File: src/main/java/com/google/codeu/data/Datastore.java (hunk 1 above)
Comment (miaboloix) [reply src/main/java/com/google/codeu/data/Datastore.java:134]: I am not sure the reasoning behind it but i know that it breaks a few things if i just convert it to an int. (that is a good question though and worth looking into more but i dont have a solid explanation) - What happens if i just save it directly as an int is i get a 'java.lang.ClassCastException: java.lang.Long cannot be cast to java.lang.Integer' error but the way i did it above is fine for some reason... even though level is never saved as a long type.

File: src/main/java/com/google/codeu/servlets/MessageFeedServlet.java (hunk 2 above)
Comment (miaboloix) [reply src/main/java/com/google/codeu/servlets/MessageFeedServlet.java:33]: on my end it is 4 spaces... is that not what we are using?

Comment (Rachelware) [reply src/main/java/com/google/codeu/servlets/MessageFeedServlet.java:33]: Just line 34 looks like an indent is missing

**Review by Rachelware — APPROVED** [review]

**Pull request merged**